.github/workflows/checks.yml

```diff
@@ -2,7 +2,6 @@ name: Checks
 
 on:
   push:
-  pull_request:
```

Comment:
I think we need `pull_request` as well. Looks like CI is not running now for this pull request.

Reply:
Last time I noticed that the checks were launched twice: for push and for pull request. I don't know why they aren't launched now.

Reply:
I set it to `push` only on `main` so there should be no duplicate launches.